web/src/components/profile/ProfileStepProgress.module.css

```diff
@@ -0,0 +1,166 @@
+.container {
+  margin: 2rem 0;
+  padding: 1.5rem;
+  background: var(--background);
+  border-radius: 8px;
+  border: 1px solid #e5e7eb;
+}
+
+.step-list {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.step {
+  position: relative;
+  padding-bottom: 2rem;
+}
+
+.step:last-child {
+  padding-bottom: 0;
+}
+
+.step-content {
+  display: flex;
+  align-items: flex-start;
+  gap: 1rem;
+}
+
+.step-icon {
+  flex-shrink: 0;
+  width: 2.5rem;
+  height: 2.5rem;
+  border-radius: 50%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-weight: 600;
+  font-size: 0.875rem;
+  position: relative;
+  z-index: 1;
+}
+
+.step-icon-pending {
+  background-color: #fef2f2;
+  color: #b91c1c;
+  border: 2px solid #fecaca;
+}
+
+.step-icon-current {
+  background-color: #0e8d3d;
```

Comment:
this might just be personal preference, but all green to me reads as everything is done, so maybe having current as red would make it more obvious it's active but not done?
onboard with pending as grey instead of red 👍

Reply:
Yeah that could work, the ticks on the 'done' icons does differentiate it with 'current'. But Maybe a different colour could work. We could have green for 'done', amber for 'current', and red for 'pending'?

Reply:
+1 for grey instead of red